ddl/column_change_test.go

```diff
@@ -152,31 +155,33 @@ func (s *testColumnChangeSuite) TestColumnChange(c *C) {
 	}
 	d.SetHook(tc)
 	defaultValue := int64(3)
-	job := testCreateColumn(c, ctx, d, s.dbInfo, tblInfo, "c3", &ast.ColumnPosition{Tp: ast.ColumnPositionNone}, defaultValue)
-	c.Assert(checkErr, IsNil)
-	testCheckJobDone(c, d, job, true)
+	job := testCreateColumn(t, ctx, d, s.dbInfo, tblInfo, "c3", &ast.ColumnPosition{Tp: ast.ColumnPositionNone}, defaultValue)
+	require.NoError(t, checkErr)
+	testCheckJobDone(t, d, job, true)
 	mu.Lock()
 	tb := publicTable
 	mu.Unlock()
-	s.testColumnDrop(c, ctx, d, tb)
-	s.testAddColumnNoDefault(c, ctx, d, tblInfo)
+	s.testColumnDrop(t, ctx, d, tb)
+	s.testAddColumnNoDefault(t, ctx, d, tblInfo)
 }
 
-func (s *testColumnChangeSuite) TestModifyAutoRandColumnWithMetaKeyChanged(c *C) {
+func TestModifyAutoRandColumnWithMetaKeyChanged(t *testing.T) {
```

Comment:
For example, this file is not under `ddl/table`. If reviewers found a 1000 lines diff including non trivial change, most of them will simply give up to review.

Take a look at https://github.com/pingcap/tidb/issues/26022 and see that we either run parallel tests, or serial test in `xxx_serial_test.go`.